### PR TITLE
Update VMM SRG mappings

### LINKED ISSUE #3871: IA-5 (1) (b) | CCI-000195 | SRG-OS-000072-VMM-000390 |   | The VMM must require the change of at least 8 of the total number of characters when passwords are changed. |
IA-5 (1) (b) | CCI-000195 | SRG-OS-000072-VMM-000390 |   | The VMM must require the change of at least 8 of the total number of characters when passwords are changed. |

 If the VMM allows the user to consecutively reuse extensive portions of passwords, this increases the chances of password compromise by increasing the window of opportunity for attempts at guessing and brute-force attacks.The number of changed characters refers to the number of changes required with respect to the total number of positions in the current password. In other words, characters may be the same within the two passwords; however, the positions of the like characters must be different. |   | Verify the VMM requires the change of at least 8 of the total number of characters when passwords are changed.If it does not, this is a finding. | Configure the VMM to require the change of at least 8 of the total number of characters when passwords are changed. 
| CAT II
-- | -- | -- | -- | -- | -- | -- | -- | -- | --

### LINKED ISSUE #3874: IA-5 (1) (d) | CCI-000198 | SRG-OS-000075-VMM-000420 |   | The VMM must enforce 24 hours/1 day as the minimum password lifetime. |
IA-5 (1) (d) | CCI-000198 | SRG-OS-000075-VMM-000420 |   | The VMM must enforce 24 hours/1 day as the minimum password lifetime. |

 Enforcing a minimum password lifetime helps to prevent repeated password changes to defeat the password reuse or history enforcement requirement. If users are allowed to immediately and continually change their password, then the password could be repeatedly changed in a short period of time to defeat the organization's policy regarding password reuse. |   | Verify the VMM enforces 24 hours/1 day as the minimum password lifetime.If it does not, this is a finding. | Configure the VMM to enforce 24 hours/1 day as the minimum password lifetime.

 | CAT II
-- | -- | -- | -- | -- | -- | -- | -- | -- | --

### LINKED ISSUE #3875: IA-5 (1) (d) | CCI-000199 | SRG-OS-000076-VMM-000430 |   | The VMM must enforce a 60-day maximum password lifetime restriction. |
IA-5 (1) (d) | CCI-000199 | SRG-OS-000076-VMM-000430 |   | The VMM must enforce a 60-day maximum password lifetime restriction. | Any password, no matter how complex, can eventually be cracked. Therefore, passwords need to be changed periodically. If the VMM does not limit the lifetime of passwords and force users to change their passwords, there is the risk that the VMM passwords could be compromised. | 
  | Verify the VMM enforces a 60-day maximum password lifetime restriction.If it does not, this is a finding. | Configure the VMM to enforce a 60-day maximum password lifetime restriction. 

| CAT II
-- | -- | -- | -- | -- | -- | -- | -- | -- | --

### LINKED ISSUE #3958: AU-8 (1) (a) | CCI-001891 | SRG-OS-000355-VMM-001330 |   | The VMM must, for networked systems, compare internal information system clocks at least every 24 hours with a server which is synchronized to one of the redundant United States Naval Observatory (USNO) time servers or a time server designated for the appropriate DoD network (NIPRNet/SIPRNet), and/or the Global Positioning System (GPS). |
AU-8 (1) (a) | CCI-001891 | SRG-OS-000355-VMM-001330 |   | The VMM must, for networked systems, compare internal information system clocks at least every 24 hours with a server which is synchronized to one of the redundant United States Naval Observatory (USNO) time servers or a time server designated for the appropriate DoD network (NIPRNet/SIPRNet), and/or the Global Positioning System (GPS). | 

Inaccurate time stamps make it more difficult to correlate events and can lead to an inaccurate analysis. Determining the correct time a particular event occurred on a system is critical when conducting forensic analysis and investigating system events. Sources outside of the configured acceptable allowance (drift) may be inaccurate.Synchronizing internal VMM clocks provides uniformity of time stamps for VMMs with multiple system clocks and systems connected over a network.Organizations should consider endpoints that may not have regular access to the authoritative time server (e.g., mobile, teleworking, and tactical endpoints). |   | Verify the VMM, for networked systems, compares internal information system clocks at least every 24 hours with a server which is synchronized to one of the redundant United States Naval Observatory (USNO) time servers or a time server designated for the appropriate DoD network (NIPRNet/SIPRNet), and/or the Global Positioning System (GPS).If it does not, this is a finding. | Configure the VMM, for networked systems, to compare internal information system clocks at least every 24 hours with a server which is synchronized to one of the redundant United States Naval Observatory (USNO) time servers or a time server designated for the appropriate DoD network (NIPRNet/SIPRNet), and/or the Global Positioning System (GPS). | CAT II
-- | -- | -- | -- | -- | -- | -- | -- | -- | --

### PR DESCRIPTION
**Pull request by redhatrises**
- Fixes #3875
- Fixes #3874
- Fixes #3871
- Fixes #3958

### CODE REVIEW

File: linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml [hunk 1]
```diff
@@ -43,9 +43,10 @@ identifiers:
 references:
     cis: "3.6"
     cui: 3.3.7
-    disa: "160"
-    nist: AU-8(1)
+    disa: "160,1891"
+    nist: AU-8(1),AU-8(1)(a)
     nist-csf: PR.PT-1
+    vsrg: SRG-OS-000355-VMM-001330
```
Comment (shawndwells) [inline linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml]:
```suggestion
    vmmsrg: SRG-OS-000355-VMM-001330
```

File: linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_difok/rule.yml [hunk 2]
```diff
@@ -39,6 +39,7 @@ references:
     nist: IA-5(b),IA-5(c),IA-5(1)(b)
     nist-csf: PR.AC-1,PR.AC-6,PR.AC-7
     srg: SRG-OS-000072-GPOS-00040
+    vsrg: SRG-OS-000072-VMM-000390
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_difok/rule.yml]:
```suggestion
    vmmsrg: SRG-OS-000072-VMM-000390
```

File: linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml [hunk 3]
```diff
@@ -38,6 +38,7 @@ references:
     nist: AC-2(2),AC-2(3)
     nist-csf: DE.CM-1,DE.CM-3,PR.AC-1,PR.AC-4,PR.AC-6
     srg: SRG-OS-000002-GPOS-00153
+    vsrg: SRG-OS-000002-VMM-000020
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml]:
```suggestion
    vmmsrg: SRG-OS-000002-VMM-000020
```

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/rule.yml [hunk 4]
```diff
@@ -22,8 +22,10 @@ identifiers:
     cce@rhel7: 80522-6
 
 references:
+    nist: IA-5(1)(d)
     disa: "199"
     srg: SRG-OS-000076-GPOS-00044
+    vsrg: SRG-OS-000076-VMM-000430
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/rule.yml]:
```suggestion
    vmmsrg: SRG-OS-000076-VMM-000430
```

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/rule.yml [hunk 5]
```diff
@@ -22,8 +22,10 @@ identifiers:
     cce@rhel7: 80521-8
 
 references:
+    nist: IA-5(1)(d)
     disa: "198"
     srg: SRG-OS-000075-GPOS-00043
+    vsrg: SRG-OS-000075-VMM000420
```
Comment (shawndwells) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/rule.yml]:
```suggestion
    vmmsrg: SRG-OS-000075-VMM000420
```

File: linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml (hunk 1 above)
Comment (redhatrises) [reply linux_os/guide/services/ntp/chronyd_or_ntpd_specify_remote_server/rule.yml]: @shawndwells it's actually `vsrg`

File: linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_difok/rule.yml (hunk 2 above)
Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_difok/rule.yml]: @shawndwells it's actually `vsrg`

File: linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml (hunk 3 above)
Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_temp_expire_date/rule.yml]: @shawndwells it's actually `vsrg`

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/rule.yml (hunk 4 above)
Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/rule.yml]: @shawndwells it's actually `vsrg`

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/rule.yml (hunk 5 above)
Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/rule.yml]: @shawndwells it's actually `vsrg`

**Comment by shawndwells** [comment]
ack pending jenkins

**Pull request merged**